docsrc/sasl/authentication_mechanisms.rst

```diff
@@ -52,30 +67,51 @@ Not sure how to get GSSAPI going? Check out our :ref:`GSSAPI configuration guide
 GSS-SPEGNO
 ----------
 
-.. todo::
-   Content needed here
+.. note::
+   If you have more informations about `GSS-SPEGNO` let us know.
 
 KERBEROS_V4
 -----------
 
-.. todo::
-   Content needed here
+Kerberos is used within Microsoft Active Directory as the preferred Authentication Protocol.
+
+Kerberos is a computer network authentication protocol, in other words, which allows nodes communicating over a non-Transport-layer Security Mechanism to prove their identity to one another in a secure manner.
+Kerberos designers aimed primarily at a client-server model, and it provides mutual Authentication.
+
+Kerberos protocol messages replay attacks.
+
+Kerberos uses Symmetric Key Cryptography and requires a trusted third party.
+Extensions to Kerberos can provide for the use of Public Key cryptography during certain phases of authentication.
+
+Defined in :rfc:`4752`
 
 LOGIN
 -----
 
-.. todo::
-   Content needed here
+Documented in a `RFC Draft: draft-murchison-sasl-login <https://tools.ietf.org/html/draft-murchison-sasl-login>`_
+
+.. warning::
+   The LOGIN SASL mechanism is obsoleted in favor of the :ref:`MECH-PLAIN` SASL mechanism.
+
+The LOGIN SASL mechanism does not provide a security layer.
+This mechanism must not be used without adequate security protection as
+the mechanism affords no integrity nor confidentiality protection itself.
 
 NTLM
 ----
 
-.. todo::
-   Content needed here
+NTLM is a proprietary protocol developed by Microsoft.
```

Comment:
This mech is obsolete and being removed from the codebase. This section can go away.

Reply:
The PR immediately preceding yours. https://github.com/cyrusimap/cyrus-sasl/pull/736
Most likely that means version 2.2.0. No further 2.1 releases are planned.